EasyPost/Request.cs

```diff
@@ -103,7 +103,7 @@ public class Request {
                         result.AddRange(FlattenParameters(list[i], string.Concat(parent, "[", pair.Key, "][", i, "]")));
                     }
                 } else if (pair.Value is DateTime) {
-                    result.Add(new Tuple<string, string>(string.Concat(parent, "[", pair.Key, "]"), Convert.ToString(((DateTime)pair.Value).ToString("yyyy-MM-ddTHH:mm:sszzz"))));
+                    result.Add(new Tuple<string, string>(string.Concat(parent, "[", pair.Key, "]"), ((DateTime)pair.Value).ToLocalTime().ToString("yyyy-MM-ddTHH:mm:sszzz")));
```

Comment:
This could be correct, but I can't say for certain unless I see a failing test and the fact that this fixes it with the expected output.